docs/client_reference.rst

```diff
@@ -135,6 +135,22 @@ The client session supports the context manager protocol for self closing.
       requests where you need to handle responses with status 400 or
       higher.
 
+      You can also provide a coroutine which takes the response as an
+      argument and can raise an exception based on custom logic, e.g.::
+
+          async def custom_check(response):
```

Comment:
This seems to have been accidentally removed by @asvetlov in https://github.com/aio-libs/aiohttp/pull/5132/files#diff-89957958d0a4f5bec39bd0390ff523269ce26597a7c20abdc25818ce658c126bL122, along with https://github.com/aio-libs/aiohttp/pull/5132/files#diff-89957958d0a4f5bec39bd0390ff523269ce26597a7c20abdc25818ce658c126bL411.